packages/core/src/components/legend-item/legend-item.scss

```diff
@@ -0,0 +1,63 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/typography";
+@use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/components/legend-item/legend-item.variables" as legend-item-variables;
+
+:host {
+  display: block;
+  padding: units.$u1 * 0.5;
+}
+
+@include utilities.box-sizing();
+
+.sr-only {
+  @include utilities.sr-only();
+}
+
+.dso-legend-item {
+  display: flex;
+  position: relative;
+
+  &.dso-legend-item-symbol {
+    + .dso-body {
+      margin-inline-start: legend-item-variables.$symbol-width + units.$u1;
+    }
+  }
+
+  &.dso-legend-item-selectable {
+    + .dso-body {
+      margin-inline-start: legend-item-variables.$selectable-width;
+    }
+
+    &.dso-legend-item-symbol {
+      + .dso-body {
+        margin-inline-start: legend-item-variables.$symbol-width + units.$u1 + legend-item-variables.$selectable-width;
+      }
+    }
+  }
+
+  button {
```

Comment:
In de basis lijkt dit op een Button Tertiary. Heb je overwogen om deze button op basis van die mixins te maken?

Reply:
✅